cluster/charts/rook-ceph/templates/resources.yaml

```
@@ -2093,6 +2093,14 @@ spec:
                         type: string
                       startTime:
                         type: string
+            quotas:
+              properties:
+                enabled:
+                  type: boolean
+                maxBytes:
+                  type: integer
```

How about setting the min 0 for each of the integers in the schema?
```suggestion
                  type: integer
                  minimum: 0
```

---

I was unaware of minimum, thanks for the tip. Change made.